<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="gestionVol">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>233</y>
     <width>319</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GESTION Donateurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gestionBillet">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>286</y>
     <width>319</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GESTION Don</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gestionBenevole">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>127</y>
     <width>319</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GESTION BENEVOLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gestionPasager">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>319</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GESTION Mamans</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
